File: T/T10/buffer.overflow_files/buffer.overflow_files/Linked_List_py/linked_driver.py
'''simple driver for LList'''

from LList import LList

print( "Creating linked-list of characters starting with ->h->e->l->l->0->NULL")
testingList = LList("hello")
print("Finding the 4th item in the list: ")
print( testingList.__getitem__(4))
print("alternative coding for finding the 4th item in the list: ")
print(testingList[4])
print("Inserting '!' in the fifth position: ")
testingList.insert(5, '!')
print(testingList[5])

File: T/T10/buffer.overflow_files/buffer.overflow_files/Linked_List_cpp/linked_driver.cpp
#include <iostream>
#include "LList_partial.cpp"
using namespace std;

int main()
{
    cout << "Creating linked-list of characters starting with ->h->e->l->l->0->NULL"<<endl;
    LList testingList = LList("hello");
    cout << "Finding the 4th item in the list: "<<endl;
    cout << testingList.__getitem__(4)->item << endl;
    cout << "Inserting '!' in the fifth position: "<<endl;
    testingList.insert(5, '!');
    cout << testingList.__getitem__(5)->item << endl;
    return 0;
}
